scripts/tools/multiMetricValues.yml

```diff
@@ -32,6 +32,7 @@ class: $tMetricGroup
 name: CalculateMultipleMetricValuesExp
 decl: static
 details:
+    - "1.6": "Note: This extension is deprecated and has now been promoted to $tMetricGroupCalculateMultipleMetricValues. It is expected to be removed on next minor version update."
```

Comment:
caution: removing experimental extensions may cause compatibility issues in DDI tables.

Reply:
@bmyates we are not removing it here but it will be eventually. This is an experimental extension, so there are no guarantees on backwards compatibility.

Reply:
Yes, understood.  However removing it will cause issues with DDI table and break compatibility of other experimental APIs. There will need to be some workarounds put into code generation to avoid this.   It may be easier to not remove it. 

https://github.com/intel-innersource/libraries.compute.oneapi.level-zero.loader/blob/21acb8b405a839e7ff09bd2a49472b217ba871a8/include/zet_ddi.h#L361